src/index.css

```diff
@@ -1,21 +1,137 @@
+:root {
+  --Dark-Blue: hsl(210, 100%, 50%);
+  --Blue: rgba(46, 46, 248, 0.754);
+  --Dark-Gray: rgb(72, 72, 72);
+  --Very-Light-Gray: hsl(0, 0%, 98%);
+  --White: hsl(0, 0%, 100%);
+}
+
+.dark-blue-color {
+  color: var(--Dark-Blue);
+}
```

Comment:
Great job @stephnna 🤝

- [ ] [Optional] Kindly add space between elements with these two classes. `redux-book-list-title` and `dark-blue-color`.